subsys/dfu/boot/mcuboot.c

```diff
@@ -96,6 +98,7 @@ struct mcuboot_v1_raw_header {
 				  BOOT_MAX_ALIGN)
 #define MAGIC_OFFS(bank_area) ((bank_area)->fa_size - BOOT_MAGIC_SZ)
 
+#ifndef CONFIG_MCUBOOT
```

Comment:
Why not then use:
```
#ifdef CONFIG_BOOTLOADER_MCUBOOT
```
so that we don't depend on a Kconfig option that is outside the Zephyr tree?